src/components/MoneyRequestConfirmationListFooter.tsx

```diff
@@ -75,6 +75,9 @@ type MoneyRequestConfirmationListFooterProps = {
     /** The distance of the transaction */
     distance: number;
 
+    /** The raw numeric amount of the transaction */
+    iouAmount: number;
```

Comment:
I'm not super pumped about this var name, is this the naming convention we normally use? I feel like it would be much clearer as something like `rawAmount`, what do y'all think? 🤔